doc/enterprise/read_views/c_api.rst

```diff
@@ -0,0 +1,274 @@
+.. _read_views_c_api:
+
+C API
```

Comment:
Perhaps we need more specific names for **C API** and **Lua API** pages. Now they appear first in search results for "C API" and "Lua API" queries across all Tarantool docs, which doesn't seem right.

Reply:
Definitely. Will rename to `Read views: Lua API` and `Read views: C API` for now. But, I hope they will be moved to `box` reference at some moment.